proto/storage/compliance_operator_v2.proto

```diff
@@ -43,7 +43,7 @@ message ComplianceOperatorProfileV2 {
     // The primary key is name-profile_version as that is guaranteed unique in the operator and how
     // the profile is referenced in scans and settings
     string              id               = 1 [(gogoproto.moretags) = 'sql:"pk"'];
-    string              profile_id       = 2;
+    string              profile_id       = 2 [(gogoproto.moretags) = 'search:"Compliance Profile ID,hidden,store"'];
```

Comment:
No need to have the store anymore. though I can submit a PR to remove all the uses. This was for the previous indexer